spec/features/create_new_work_spec.rb

```diff
@@ -117,14 +117,25 @@
           fill_in 'Abstract', with: 'User provided abstract'
           check 'Oral history'
 
+          find('label.switch').click # Use auto-generated citation
+          fill_in 'Provided citation', with: 'Citation from user input'
+
           click_button 'Deposit'
 
+          # Check keyword validation: at least one and no duplicates
           expect(page).to have_content('Keyword must be filled in')
 
-          fill_in 'Keyword', with: 'Springs'
+          # add the same keyword twice
+          all('#keyword-input-field')[0].set 'Springs'
+          click_button '+ Add another keyword'
+          all('#keyword-input-field')[1].set 'Springs'
 
-          find('label.switch').click # Use auto-generated citation
-          fill_in 'Provided citation', with: 'Citation from user input'
+          click_button 'Deposit'
```

Comment:
just moved this citation setting block up a bit so that all the keyword checking code is grouped together...order doesn't matter in this case